docs: advise using fewer threads if OOM while compiling (#5053)
yugabyte/yugabyte-db: docs/content/latest/contribute/core-database/build-from-src-centos.md

./yb_build.sh release
```

{{< note title="Note" >}}

> If you are seeing errors such as `g++: internal compiler error: Killed`, it's probably because the system has ran out of memory.

Reviewer: ```suggestion
If you see errors, such as `g++: internal compiler error: Killed`, the system has probably run out of memory.
```

Try again by running the build script with less concurrency `-j1`.

{{< /note >}}

> The above command will build the release configuration, put the C++ binaries in `build/release-gcc-dynamic-ninja`, and will also create the `build/latest` symlink to that directory.

Reviewer: ```suggestion
The command above will build the release configuration, add the C++ binaries into the `build/release-gcc-dynamic-ninja` directory, and create a `build/latest` symlink to that directory.
```

./yb_build.sh release
```

{{< note title="Note" >}}

If you see errors, such as `g++: internal compiler error: Killed`, the system has probably run out of memory.

> Try again by running the build script with less concurrency `-j1`.

Reviewer 2: with less concurrency
-->
with less concurrency, for example, `-j1`.
